network-area-diagram-viewer/src/main/java/com/powsybl/nad/viewer/Model.java

```diff
@@ -81,8 +107,11 @@ public void setSvgContent(String svgContent) {
 
     public SvgParameters getSvgParameters() {
         return new SvgParameters()
-                .setSvgWidthAndHeightAdded(true)
-                .setEdgeInfoAlongEdge(infoAlongEdge.get());
+                .setIdDisplayed(idDisplayed.get())
+                .setInsertNameDesc(insertNameDesc.get())
+                .setSubstationDescriptionDisplayed(substationDescriptionDisplayed.get())
```

Comment:
the multiline does not seem to work with javafx webview

Reply:
nope; the SVG is displayed ok in the browser (Chrome), but the javafx paints substation name overlapped with voltage level name. I will try to change the `y` position of the text span, instead of only `dy`.

Reply:
the multiline works in javafx webview, but only for _reasonable_ font sizes: we are specifying a very small font size (`0.25px`). It seems the viewer rounds the size to nearest integer for some calculations (although not when calculating `em` spacing). For sizes above `1px` the `dy` works and the space between spans is appropriate at `1.1em`. For sizes below `1px` the space between text spans is bigger than expected, for font size `< 0.5px` we have overlap.

with font size `0.5px`:
<img width="260" alt="font_size_0 5px" src="https://user-images.githubusercontent.com/3374819/200513924-e2013784-918e-4fda-9d80-080b3e8013d9.png">

with font size `0.75px`:
<img width="252" alt="font_size_0 75px" src="https://user-images.githubusercontent.com/3374819/200513926-44f7f679-c0aa-477b-aec5-415a8855f47b.png">

and with font size `1px`:
<img width="358" alt="font_size_1px" src="https://user-images.githubusercontent.com/3374819/200513927-689ce148-92d9-4f37-97eb-eab379c5fe02.png">

Reply:
I think we should adjust the coordinates in the SVG space to have _reasonable_ sizes for fonts in pixels (also stroke widths have very small numbers in the style sheet). Is there any other reason to have size of drawings so _small_ ?

Reply:
ok, thanks for investigating this! one more reason to have *reasonable* sizes indeed. In fact I needed it for another reason last week, let's do this! I'll open a PR on that.